share/spice/quandl/fundamentals/fundamentals_triggers.txt

```diff
@@ -0,0 +1,54 @@
+total assets	ASSETS
```

Comment:
Unfortunately, no.  The code takes ordering into account to give preference to more specific phrases.  So a search with "current assets" will return the ASSETSC identifier, whereas simply searching with "assets" will return ASSETS.  If the list were alphabetized, the "assets" line would trigger even if people searched on "current assets".